tests/integration/unit_test/test_unit_test_go1_x.py

```diff
@@ -10,6 +10,10 @@ class UnitTest_provided_go_cookiecutter_aws_sam_hello_golang(UnitTestBase.GoUnit
     directory = "al2/go/hello"
     code_directories = ["hello-world"]
 
+class UnitTest_providedal2023_go_cookiecutter_aws_sam_hello_golang(UnitTestBase.GoUnitTestBase):
```

Comment:
we need to add unit test case for the go/hello-img

Reply:
it seems like we don't do unit test for hello-img for any runtime. We only do the "build_invoke" test for hello-img for all runtimes

Reply:
also it seems that sam init is failing for this template, not sure why

Reply:
the cookiecutter.json seems to contain incorrect values. Fixed it and now waiting for the test run